### DISCUSSION TITLE
Morphological input data units?

### DISCUSSION THREAD

**Discussion by MalveHeinz** [mhm-ufz/mHM#92] [General]
Hello,

I am just starting to set up mHM for my case study and have some basic questions. I was wondering about the units of the :
- slope layer, is it in degrees (0-90) or % rise (0-inf)? 
- aspect, its 0-360, i assume?
- flow accumulation is in number of contributing cells?

I dont really understand how the geology is incorporated. I understand that we need to identify and locate dominant classes, but what are the parameters in the mhm_parameter.nml file, that align with those classes, representing ? Are the parameters only determined through model calibration?

Thanks a lot in advance!
Malve

**Comment by pallavhydro** [comment]
Hi Malve

For the `first part of your question`:

If you use GIS, then I would recommend you to follow [this section](https://mhm.pages.ufz.de/mhm/stable/data.html#morph_data) of the mhm documentation, for the preparation of morphological input as well as to understand the units.

Following are the units used:
- slope: percent rise.
- aspect: 0-360 indicating azimuth (i.e. North)
- flow accumulation: yes its number of contributing cells

For the `second part of your question`:

This is something that would be useful to include in the mhm documentation. What do you think @MuellerSeb @StephanThober ?
@rohinikumar82 @StephanThober could you please help with the second part of Malve's question. TIA.

Cheers
Pallav

**Reply by rakovec** [reply]
Regarding point #2 

Exactly, the GeoParam are defined through calibration, to match baseflow components.  Geology is not yet regionalized in mHM. Geological classes are only used to defined properties on the saturated (~groundwater reservoir) with linear outflow. As such, each individual geological class is allowed to have it's own "time constant" (units in days, default value is 100, bounds from 1 to 1000 for the calibration case).

**Reply by MalveHeinz** [reply]
Thanks a lot for the fast reply and explanation @pallavhydro and @rakovec !

**Reply by pallavhydro** [reply]
Hi @MalveHeinz 

If the discussion here solved your issue, I'd appreciate you mark the response that solved the issue.

Thanks in advance.
Pallav
